cachalot/tests/transaction.py

```diff
@@ -187,3 +189,13 @@ def test_deferred_error(self):
                         '-- ' + Test._meta.db_table)  # Should invalidate Test.
         with self.assertNumQueries(1):
             list(Test.objects.all())
+
+
+class AtomicCacheTestCase(SimpleTestCase):
+    def setUp(self):
+        self.atomic_cache = AtomicCache(cache, 'db_alias')
+    
+    def test_set(self):
```

Comment:
Is there a reason for this test case?

Reply:
> Is there a reason for this test case?

@Andrew-Chen-Wang it’s to improve test coverage. But we might also check if the `AtomicCache.set` method is used. If not, we could remove it?